app/move/controllers/update/document.js

```diff
@@ -0,0 +1,59 @@
+const { get } = require('lodash')
+
+const moveService = require('../../../../common/services/move')
+const DocumentUploadController = require('../create/document')
+
+const UpdateBase = require('./base')
+
+class UpdateDocumentUploadController extends UpdateBase {
+  configure(req, res, next) {
+    const canUploadDocuments = get(
+      req,
+      'session.currentLocation.can_upload_documents'
+    )
+    if (!canUploadDocuments) {
+      const error = new Error(
+        'Document upload is not possible for this location'
+      )
+      error.statusCode = 404
+      return next(error)
+    }
+    DocumentUploadController.prototype.configure.apply(this, [req, res, next])
+  }
+
+  getUpdateValues(req, res) {
+    const move = req.getMove()
+    if (!move) {
+      return {}
+    }
+    const values = {}
+    if (req.initialStep) {
+      req.sessionModel.set('documents', move.documents)
+    }
+    values.documents = req.sessionModel.get('documents')
```

Comment:
Doesn't the parent getValues do something like retrieving this? Wonder if we can re-use that across all the update controllers.

Reply:
Yes, now it's all in place, we can probably streamline this. As part of the move to sticking everything on the `req` object